src/Reflex/Dom/SemanticUI/Button.hs

```diff
@@ -124,14 +137,16 @@ uiButtonAttrs UiButton{..} = T.unwords $ catMaybes
 -- provided by other functions such as 'uiButtonAnimated'.
 uiButton
     :: MonadWidget t m
-    => Dynamic t UiButton
+    => UiButtonType
```

Comment:
I thought of that but it seems very odd to change the type of a button dynamically.